packages/driver/cypress/support/index.d.ts

```diff
@@ -0,0 +1,5 @@
+/// <reference types="chai" />
+/// <reference types="chai-subset" />
+/// <reference types="cypress" />
+
+/// <reference path="../../types/internal-types.d.ts" />
```

Comment:
I have a feeling there's something extra I need to do to get our internal types in our cypress tests. Sometimes they work and sometimes they don't though.